ocs_ci/utility/metadata_utils.py

```diff
@@ -0,0 +1,308 @@
+"""
+Module that contains all operations related to add metadata feature in a cluster
+
+"""
+
+import logging
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import (
+    CommandFailed,
+    ResourceWrongStatusException,
+)
+
+log = logging.getLogger(__name__)
+
+
+def check_setmetadata_availability(pod_obj):
+    """
+    Check setmetadata parameter is available or not for cephfs and rbd plugin pods
```

Comment:
You are getting all the cephfs and rbd plugin pods inside the function, what is the use of this pod_obj?
and please add doc string for this argf

Reply:
Need to check for each of the plugin pods setmetadata parameter is available when metadata feature is enabled. So to run the oc command have used the pod_obj.